packages/sdks/src/components/content-variants/content-variants.lite.tsx

```diff
@@ -37,7 +38,7 @@ type VariantsProviderProps = ContentVariantsPrps & {
    * For internal use only. Do not provide this prop.
    */
   isNestedRender?: boolean;
-};
+} & CustomApiBaseUrl;
```

Comment:
to keep the types of `Content` consistent, you should add `apiBaseUrl` here: 

https://github.com/BuilderIO/builder/blob/main/packages/sdks/src/components/content-variants/content-variants.types.ts#L35


then you shouldn't need to add `& CustomApiBaseUrl` everywhere